typescript/packages/subsurface-viewer/src/layers/axes2d/axes2DLayer.ts

```diff
@@ -53,13 +54,57 @@ enum ViewSide {
     Top,
 }
 
+function MakeExpLabel(a: number): string {
+    let label = d3.format(".2e")(a);
+    label = label.replace("e", "E");
+    label = label.replace("\u2212", "-");
+    return label;
+}
+
+function UseExponential(
```

Comment:
I don't think it's useful to condition notation on the space it takes, but rather a fixed rule, eg. scientific notation if number of digits > 6.

Reply:
I changed it to a fixed rule but so that utm coordinates shows up in non-exponential (digits > 7). Otherwise there may be several ticks with the same exponential number.

Reply:
Ok, makes sense. The `UseExponential` still seems a bit magic to me. Maybe there should be some unit tests to cover it?